inst/pages/exercises.qmd

```diff
@@ -1552,6 +1552,24 @@ sechm(
     show_colnames=TRUE)
 ```
 
+5. Create a basic heatmap of the data using the `pheatmap` package.
+
+```{r}
+#| label: heatmap-ex5
+#| code-fold: true
+#| code-summary: "Show solution"
+#| eval: false
+
+library(pheatmap)
+
+# Create example data
+set.seed(123)
+example_data <- matrix(rnorm(200), nrow = 20, ncol = 10)
+
+# Create basic heatmap using pheatmap
+pheatmap(example_data)
```

Comment:
We should use the ready made demo data sets from real studies (included in mia, miaViz, miaTime packages).